src/components/KanbanColumn/KanbanColumn.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from 'react';
+import KanbanCard from '../KanbanCard/KanbanCard';
+
+
+class KanbanColumn extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    const cardList = this.props.cards;
+
+    const inQueue = cardList.filter((card) => {
```

Comment:
You only need to filter the cards once here. You want to filter by the `this.props.status` that you're passing into this component from KanbanBoard